### PR TITLE
feat: add block option that's return non-zero if skipped

### PR DESCRIPTION
**Pull request by d0riven**
```
⟩ dist/blaker -r ap-northeast-1 echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`

⟩ echo $status
0

⟩ dist/blaker -r ap-northeast-1 -b echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`

⟩ echo $status
250
```

### CODE REVIEW

File: pkg/blaker/blaker.go [hunk 1]
```diff
@@ -45,15 +45,11 @@ func (b *Blaker) RunCmd(input *RunCmdInput) (cmd.Status, error) {
...
-		msg := fmt.Sprintf("the command cannot be run after %s. skipped command: `%s %s`",
-			breakTime.Format(time.RFC3339),
-			input.Command,
-			strings.Join(input.Args, " "),
-		)
-		if _, err := fmt.Fprintf(input.Stderr, msg); err != nil {
-			return cmd.Status{}, errors.Wrapf(err, "failed to write skipped log: %s", msg)
+		skipErr := NewSkipError(*breakTime, input)
+		if _, err := fmt.Fprintf(input.Stderr, skipErr.Error()); err != nil {
+			return cmd.Status{}, errors.Wrapf(err, "failed to write skipped log: %s", skipErr.Error())
```
Comment (cynipe) [inline pkg/blaker/blaker.go]: I think it would be nicer not to write message here but in caller side.

Comment (d0riven) [reply pkg/blaker/blaker.go]: fix https://github.com/cynipe/blaker/pull/3/commits/6ca8f587a162e5170aacfa3843a5c1a1e6e9e288

**Comment by d0riven** [comment]
```
⟩ dist/blaker -r ap-northeast-1 echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`

⟩ echo $status
0


⟩ dist/blaker -r ap-northeast-1 -E echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`
skipped error occurred


⟩ echo $status
250

```

File: pkg/blaker/errors.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package blaker
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type SkipError struct {
```
Comment (cynipe) [inline pkg/blaker/errors.go]: better to name BreakError?

File: cmd/blaker/main.go [hunk 3]
```diff
@@ -82,8 +89,24 @@ func run(ctx *cli.Context) error {
 		Stderr:  ctx.App.ErrWriter,
 	})
 
+	return errorHandling(err, ctx, status)
+}
+
+func errorHandling(err error, ctx *cli.Context, status cmd.Status) error {
```
Comment (cynipe) [inline cmd/blaker/main.go]: handleError

File: pkg/blaker/errors.go (hunk 2 above)
Comment (d0riven) [reply pkg/blaker/errors.go]: I think so too. 
fixed https://github.com/cynipe/blaker/pull/3/commits/48f3dd8b0c6a337d204ac6d7a29fa02fa3b53bc1

File: cmd/blaker/main.go (hunk 3 above)
Comment (d0riven) [reply cmd/blaker/main.go]: fixed https://github.com/cynipe/blaker/pull/3/commits/f9b63a2e2694e1c6fd68c74ab632df9d2f3ecaee

**Comment by d0riven** [comment]
check

```
⟩ dist/blaker -r ap-northeast-1 echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`

⟩ echo $status
0


⟩ dist/blaker -r ap-northeast-1 -E echo hey
using region: ap-northeast-1
the command cannot be run after 2020-01-07T20:42:00+09:00. skipped command: `echo hey`

⟩ echo $status
250
```

**Comment by cynipe** [comment]
thanks

**Pull request merged**